client/state/sites/plans/test/selectors.js

```diff
@@ -53,6 +55,74 @@ describe( 'selectors', () => {
 			expect( plans ).to.eql( plans1 );
 		} );
 	} );
+	describe( '#getCurrentPlan()', () => {
+		context( 'when no plan data is found for the given siteId', () => {
+			const siteId = 99999999;
+			const state = {
+				sites: {
+					plans: {
+						[ siteId ]: {}
+					}
+				}
+			};
+
+			it( 'returns null', () => {
+				const plan = getCurrentPlan( state, siteId );
+				expect( plan ).to.eql( null );
+			} );
+		} );
+
+		context( 'when plans are found for the given siteId', () => {
+			const siteId = 99999999;
+			const plan1 = { currentPlan: true };
+			const state = {
+				sites: {
+					plans: {
+						[ siteId ]: {
+							data: [ plan1 ]
+						}
+					},
+					items: {
+						[ siteId ]: {
+							URL: 'https://example.wordpress.com'
+						}
+					}
+				}
+			};
+
+			context( 'when those plans include a \'currentPlan\'', () => {
+				before( () => {
+					plan1.currentPlan = true;
```

Comment:
Let's avoid mutating test data. For readability it's easier to read when it's defined close to the actual test. (In tests it's ok to be redundant )